base/float.jl

```diff
@@ -464,6 +464,20 @@ round(x::IEEEFloat, ::RoundingMode{:Down})    = floor_llvm(x)
 round(x::IEEEFloat, ::RoundingMode{:Up})      = ceil_llvm(x)
 round(x::IEEEFloat, ::RoundingMode{:Nearest}) = rint_llvm(x)
 
+rounds_up(x, ::RoundingMode{:Down}) = false
+rounds_up(x, ::RoundingMode{:Up}) = true
+rounds_up(x, ::RoundingMode{:ToZero}) = x < 0
+rounds_up(x, ::RoundingMode{:FromZero}) = x > 0
```

Comment:
```suggestion
rounds_up(x, ::RoundingMode{:ToZero}) = signbit(x)
rounds_up(x, ::RoundingMode{:FromZero}) = !signbit(x)
```
I suppose zero is never rounded, in which case I don't think this matters. But if these get used in some other functionality later, `signbit` may be the more robust option.

Reply:
signbit is also probably faster in some cases.